tmt/export/rst.py

```diff
@@ -13,10 +13,10 @@ def export_story(cls,
                      story: tmt.base.Story,
                      keys: Optional[List[str]] = None,
                      include_title: bool = True) -> str:
-        template_filepath = os.path.join(tmt.export.TEMPLATES_DIRECTORY, 'default-story.rst.j2')
-
-        return tmt.utils.render_template_file(
-            template_filepath,
+        return tmt.export.template.TemplateExporter.render_template(
+            None,
```

Comment:
Let's use keyword params for all parameters here to make it clear what is `None`.

Reply:
Fixed in 2a807b73